nbformat/validator.py

```diff
@@ -235,29 +235,41 @@ def better_validation_error(error, version, version_minor):
     return NotebookValidationError(error, ref)
 
 
-def validate(nbjson, ref=None, version=None, version_minor=None, relax_add_props=False):
-    """Checks whether the given notebook JSON conforms to the current
-    notebook format schema.
+def validate(nbdict=None, ref=None, version=None, version_minor=None,
+             relax_add_props=False, nbjson=None):
+    """Checks whether the given notebook dict-like object
+    conforms to the current notebook format schema.
+
 
     Raises ValidationError if not valid.
     """
+
+    # backwards compatibility for nbjson argument
+    if nbdict is not None:
+        pass
+    elif nbjson is not None:
+        nbdict = nbjson
+    else:
+        raise ValidationError("Object to validate was not provided.")
```

Comment:
I would make this a `TypeError` rather than a ValidationError, because that's what Python normally does if you don't pass a required parameter:

```
 1│ def f(x): pass

 2│ f()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-c43e34e6d405> in <module>()
----> 1 f()

TypeError: f() missing 1 required positional argument: 'x'
```